### DISCUSSION TITLE
Does Elsa 3.0 have examples of implementing various activities in the designer? Thank you!

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5618] [General]
Does Elsa 3.0 have examples of implementing various activities in the designer? Thank you!

**Comment by glime-ai** [comment]
As of my knowledge cutoff date, the official Elsa documentation and GitHub repository do not provide specific examples of implementing various activities in the designer for Elsa 3.0. However, the documentation does provide some general guidance on how to define workflows programmatically, and it mentions that Elsa comes with a visual designer that allows you to define workflows.

For more detailed examples and guidance on using the Elsa designer, you may need to refer to community resources, such as blog posts, forums like Stack Overflow, or the official Elsa GitHub repository for potential community-contributed examples. Additionally, you can explore the Elsa Dashboard, which is a part of Elsa 3.0, to experiment with creating workflows using the designer interface.

If you are looking for specific examples or tutorials, I would recommend checking the official Elsa GitHub repository for any updates or examples that may have been added after my knowledge cutoff date, or reaching out to the Elsa community for assistance.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
